Jenkinsfile

```diff
@@ -770,26 +771,27 @@ pipeline {
                         }
                     }
                 } // stage('Scan Leap 15 RPMs')
-                stage('Fault injection testing') {
+                stage('Fault injection testing on EL 8') {
                     when {
                         beforeAgent true
                         expression { ! skipStage() }
                     }
                     agent {
                         dockerfile {
-                            filename 'utils/docker/Dockerfile.centos.7'
+                            filename 'utils/docker/Dockerfile.el.8'
                             label 'docker_runner'
                             additionalBuildArgs dockerBuildArgs(repo_type: 'stable',
                                                                 parallel_build: true,
-                                                                deps_build: true)
+                                                                deps_build: true) +
+                                                ' --build-arg DAOS_KEEP_SRC=yes'
```

Comment:
The dockerfiles should all be similar in the handling of this, if DAOS_KEEP_SRC is required for el8 but not el7 then it's probably better to update the dockerfile.

Reply:
I am not sure that the DAOS_KEEP_SRC is still needed after making the other changes.  I need to test that.